app/Http/Controllers/Device/Tabs/PortsController.php

```diff
@@ -373,6 +373,11 @@ private function getFilteredPortsQuery(Device $device, array $relationships = []
             ->when(! $this->settings['ignored'], fn (Builder $q, $disabled) => $q->where('ignore', 0))
             ->when($this->settings['admin'] != 'any', fn (Builder $q, $admin) => $q->where('ifAdminStatus', $this->settings['admin']))
             ->when($this->settings['status'] != 'any', fn (Builder $q, $admin) => $q->where('ifOperStatus', $this->settings['status']))
+            ->when($this->settings['sort'] == 'port', fn (Builder $q, $sort) => $q
+                ->orderByRaw('SOUNDEX(ifName)' . $this->settings['order'])
```

Comment:
Is there supposed to be a space here?  

What types of ifnames were tested?

Reply:
> Is there supposed to be a space here?

yes, you re right
my mistake